lib/version_utils.pm

```diff
@@ -194,6 +195,16 @@ sub check_version {
     croak "Unsupported version parameter for check_version: '$query'";
 }
 
+=head2 is_microos
+
+Check if product is SUSE MicroOS
+Version: 5.X, 6.X
```

Comment:
So it was 6.x for opensuse and 5.x for sle?

Reply:
No. For openSUSE `VERSION` is `15.2` (Leap) or `Tumbleweed` for TW. 
Enterprise version will be `5.X` and so on.

Reply:
Thx for clarification!